proto/proxy/accept_policy/accept_policy.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package conduit.proxy.accept_policy;
+
+import "common/common.proto";
+
+service AcceptPolicy {
+  // Describes how a proxy should accept inbound connections from remote peers.
+  //
+  // Includes;
+  // - Protocol detection
+  // - [TODO] TLS termination
+  rpc Inbound(InboundAcceptPolicyRequest) returns (stream InboundAcceptPolicy) {}
+
+  // Describes how a proxy should accept outbound connections from local peers.
+  //
+  // Includes;
+  // - Protocol detection
+  rpc Outbound(OutboundAcceptPolicyRequest) returns (stream OutboundAcceptPolicy) {}
+}
+
+message InstanceId {
+  // TODO
+}
+
+message InboundAcceptPolicyRequest {
+  InstanceId instance_id = 1;
+}
+
+// Describes how an inbound proxy should accept connections and requests to be routed.
+//
+// Inbound proxies may terminate TLS as they accept a connection.
+message InboundAcceptPolicy {
+  repeated EndpointMatch detection_disabled = 1;
```

Comment:
detection disabled means proxy as TCP, right?  I wonder if this can be named to make that more explicit.  The current name invites the question: "If we don't do detection then what DO we do?"